### DISCUSSION TITLE
How can I change the displayed value for the label of select dropdown after an option is selected from the dropdown list?

### DISCUSSION THREAD

**Discussion by daksh-sagar** [mantinedev/mantine#3538] [Q&A] [answered]
I have a dropdown with large labels (like this - `{label: 'some large searchable label here', value: 'some value'}`).

When the user selects an option from the dropdown i want to show only a part of the label and not the entire label. Is there a way i can do this?

**Comment by rtivital** [comment, marked as answer]
You can swap the item in the data array when user selects it. You can also use custom item component to use different properties to display value and item.

**Comment by daksh-sagar** [comment]
awesome, thanks for the response. I've done this by using custom item component :) .
